src/dnx.clr/dnx.clr.vcxproj

```diff
@@ -21,8 +21,8 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{0E4CB542-5E58-4D24-9CAE-DAC83D932DBE}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <RootNamespace>krenet45</RootNamespace>
-    <ProjectName>kre.clr</ProjectName>
+    <RootNamespace>dnxnet45</RootNamespace>
```

Comment:
I realize RootNamespace is mostly irrelevant in a C++ project, but we should correct this value while we're here.